@xen-orchestra/web-core/lib/components/state-hero/VtsErrorNoDataHero.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <VtsStateHero class="vts-error-hero" image="error-no-data" :type>{{ $t('error-no-data') }}</VtsStateHero>
+</template>
+
+<script lang="ts" setup>
+import VtsStateHero, { type StateHeroType } from '@core/components/state-hero/VtsStateHero.vue'
+
+defineProps<{
+  type: StateHeroType
+}>()
+</script>
+
+<style scoped lang="postcss">
+.vts-error-hero {
+  background-color: var(--color-danger-background-selected);
+
+  :last-child {
+    color: var(--color-danger-txt-base);
+  }
+}
+</style>
```

Comment:
If the error color is handled in the main `VtsStateHero`, this would not be necessary.
```suggestion
```